src/crypto/CHIPCryptoPAL.cpp

```diff
@@ -141,19 +141,19 @@ CHIP_ERROR Spake2p::BeginVerifier(const unsigned char * my_identity, size_t my_i
     error = PointLoad(Lin, Lin_len, L);
     VerifyOrExit(error == CHIP_NO_ERROR, error = CHIP_ERROR_INTERNAL);
 
-    state = CHIP_SPAKE2P_STATE_STARTED;
-    role  = CHIP_SPAKE2P_VERIFIER;
+    state = CHIP_SPAKE2P_STATE::CHIP_SPAKE2P_STATE_STARTED;
+    role  = CHIP_SPAKE2P_ROLE::CHIP_SPAKE2P_VERIFIER;
```

Comment:
Again, feels like values could be shorter now.  Again, followup, probably separate from the state one.